src/components/ChatEntry.js

```diff
@@ -1,22 +1,36 @@
 import React from 'react';
+// import TimeStamp from './TimeStamp';
 import './ChatEntry.css';
 import PropTypes from 'prop-types';
 
 const ChatEntry = (props) => {
+  const heart = props.liked ? '🤍'  : '❤️';
+  console.log(heart)
```

Comment:
We want to delete logging statements and other diagnostic code before opening PRs to ensure we don't bring them into the final codebase.